### ISSUE TITLE
bug: issues with some formes (radical red)

### ISSUE THREAD

**Issue by andrewbenington** [andrewbenington/OpenHome#91]
Radical Red forme/species conversion is incorrect for certain formes

non-exhaustive list:

- Shellos/Gastrodon (East Sea)
- All minior formes
- All ogerpon (besides teal mask)
- All flabebe/floette/florges (besides red)
- All scatterbug/spewpa/vivillon (besides icy snow)
- All squawkabilly (besides green)
- Female formes for:
  - Meowstic
  - Basculegion
  - Indeedee
  - Oinkologne

![Screenshot 2024-12-21 at 6 08 04 PM](https://github.com/user-attachments/assets/033e6183-2aed-4a83-a1a1-8ebdb9a2799a)

@evanwporter

**Comment by evanwporter** [comment]
These forms don't appear in RR. Will need to add these all to the transfer restrictions I guess.

**Comment by andrewbenington** [comment]
@evanwporter sprites exist for at least some of them though, i would be surprised if they didn't exist in the game. Some examples:
![Screenshot 2024-12-21 at 7 09 12 PM](https://github.com/user-attachments/assets/c32c8a6a-e6da-4828-a8f2-8a6c868b17d4)
![Screenshot 2024-12-21 at 7 09 51 PM](https://github.com/user-attachments/assets/e6673c49-6e2f-42cc-9568-acbc3a6875a1)
![Screenshot 2024-12-21 at 7 10 08 PM](https://github.com/user-attachments/assets/417f29f3-3a50-4295-8ce2-ae33d7a27237)
![Screenshot 2024-12-21 at 7 10 57 PM](https://github.com/user-attachments/assets/b94337ba-f552-4399-b8b6-54f498e2822b)
![Screenshot 2024-12-21 at 7 11 11 PM](https://github.com/user-attachments/assets/24ca98b4-c7b1-46f5-bcce-ff00028687b2)

**Comment by evanwporter** [comment]
Found the issue for some of them.

**Comment by evanwporter** [comment]
@andrewbenington 

```
export const RRSprites = Object.fromEntries(
  Object.entries(_RRSprites).map(([key, value]) => [
    key,
    value.replace(/\w+/g, (match) => match.charAt(0).toUpperCase() + match.slice(1).toLowerCase()),
  ])
)
```

I added this line to the end of `RadicalRedSprites` also changed the name. Temporary fix but it should work for now I hope...

**Comment by evanwporter** [comment]
@andrewbenington 

Fixed this with [25d915c](https://github.com/andrewbenington/OpenHome/pull/82/commits/25d915c4d1a3b324b593f3d78ca8a7acd3368dc9) over in the unbound branch. Will be able to merge soon.
